src/app/public/modules/popover/popover.component.ts

```diff
@@ -98,6 +98,8 @@ export class SkyPopoverComponent implements OnDestroy {
 
   /**
    * Specifies the type of popover.
+   *
```

Comment:
We don't need the extra line here. Typedoc will handle the annotations here to display them correctly.